Kernel/CommandLine.cpp

```diff
@@ -59,6 +56,20 @@ UNMAP_AFTER_INIT CommandLine::CommandLine(const String& string)
     }
 }
 
+UNMAP_AFTER_INIT CommandLine::CommandLine(const String& string)
+{
+    s_the = this;
+    auto embedded_cmdline = StringView(s_embedded_cmd_line);
+    StringBuilder builder;
+    builder.append(embedded_cmdline);
+    builder.append(" ");
+    builder.append(string.substring_view(0));
```

Comment:
any reason for not just doing: `builder.append(string);` ?

Reply:
Not really, now that I see it works OK with `String` too :)